Mathlib/NumberTheory/LSeries/SumCoeff.lean

```diff
@@ -153,3 +160,222 @@ theorem LSeries_eq_mul_integral_of_nonneg (f : ℕ → ℝ) {r : ℝ} (hr : 0 
+
+include hlim hfS in
+private theorem LSeries_tendsto_sub_mul_nhds_one_of_tendsto_sum_div_aux₃ {ε : ℝ} (hε : ε > 0) :
+    ∃ C ≥ 0, ∀ s : ℝ, 1 < s → ‖(s - 1) * LSeries f s - l * s‖ ≤ (s - 1) * s * C + s * ε := by
+  obtain ⟨T, hT₁, hT⟩ := (eventually_forall_ge_atTop.mpr
+    (LSeries_tendsto_sub_mul_nhds_one_of_tendsto_sum_div_aux₁
+      hlim hε)).frequently.forall_exists_of_atTop 1
+  let S : ℝ → ℂ := fun t ↦ ∑ k ∈ Icc 1 ⌊t⌋₊, f k
+  let C := ∫ t in Set.Ioc 1 T, ‖S t - l * t‖ * t ^ (-1 - 1 : ℝ)
+  have hC : 0 ≤ C := by
+    refine setIntegral_nonneg_ae measurableSet_Ioc (univ_mem' fun t ht ↦ ?_)
+    exact mul_nonneg (norm_nonneg _) <| Real.rpow_nonneg (zero_le_one.trans ht.1.le) _
+  refine ⟨C, hC, fun s hs ↦ ?_⟩
+  have hs' : 0 ≤ (s - 1) * s := mul_nonneg (sub_nonneg.mpr hs.le) (zero_le_one.trans hs.le)
+  have h₀ : LocallyIntegrableOn (fun t ↦ S t - l * t) (Set.Ici 1) := by
+    refine .sub ?_ <| ContinuousOn.locallyIntegrableOn (by fun_prop) measurableSet_Ici
+    convert locallyIntegrableOn_mul_sum_Icc f zero_le_one (locallyIntegrableOn_const 1)
+    rw [one_mul]
+  have h₁ : IntegrableOn (fun t ↦ ‖S t - l * t‖ * t ^ (-s - 1)) (Set.Ici 1) :=
+    lemma₂ hs (zero_lt_one.trans_le hT₁) h₀ fun t ht ↦ (hT t ht).le
+  have h₂ : IntegrableOn (fun t : ℝ ↦ ‖S t - l * t‖ * (t ^ ((-1 : ℝ) - 1))) (Set.Ioc 1 T) := by
+    refine ((h₀.norm.mul_continuousOn ?_ isLocallyClosed_Ici).integrableOn_compact_subset
+      Set.Icc_subset_Ici_self isCompact_Icc).mono_set Set.Ioc_subset_Icc_self
+    exact fun t ht ↦ (Real.continuousAt_rpow_const _ _
+      <| Or.inl (zero_lt_one.trans_le ht).ne').continuousWithinAt
+  have h₃ : (s - 1) * ∫ (t : ℝ) in Set.Ioi 1, (t : ℂ) ^ (-s : ℂ) = 1 := by
+    rw [integral_Ioi_cpow_of_lt (by rwa [neg_re, neg_lt_neg_iff]) zero_lt_one, ofReal_one,
+      one_cpow, show -(s : ℂ) + 1 = -(s - 1) by ring, neg_div_neg_eq, mul_div_cancel₀]
+    exact (sub_ne_zero.trans ofReal_ne_one).mpr hs.ne'
+  let Cs := ∫ t in Set.Ioc 1 T, ‖S t - l * t‖ * t ^ (-s - 1)
+  have h₄ : Cs ≤ C := by
+    refine setIntegral_mono_on ?_ h₂ measurableSet_Ioc fun t ht ↦ ?_
+    · exact h₁.mono_set <| Set.Ioc_subset_Ioi_self.trans Set.Ioi_subset_Ici_self
+    · gcongr
+      exact ht.1.le
+  calc
+    _ = ‖((s - 1) * s * ∫ t in Set.Ioi 1, S t * ↑t ^ (-(s : ℂ) - 1)) -
+          l * s * ((s - 1) * ∫ (t : ℝ) in Set.Ioi 1, ↑t ^ (-(s : ℂ)))‖ := ?_
+    _ = ‖(s - 1) * s *
+          ∫ (t : ℝ) in Set.Ioi 1, (S t * (t : ℂ) ^ (-s - 1 : ℂ) - l * t ^ (-s : ℂ))‖ := ?_
+    _ = ‖(s - 1) * s * ∫ (t : ℝ) in Set.Ioi 1, (S t - l * t) * (t : ℂ) ^ (-s - 1 : ℂ)‖ := ?_
+    _ ≤ (s - 1) * s * ∫ (t : ℝ) in Set.Ioi 1, ‖(S t - l * ↑t) * ↑t ^ (-s - 1 : ℂ)‖ := ?_
+    _ = (s - 1) * s * ∫ (t : ℝ) in Set.Ioi 1, ‖S t - l * t‖ * t ^ (-s - 1) := ?_
+    _ = (s - 1) * s * (Cs + ∫ (t : ℝ) in Set.Ioi T, ‖S t - l * t‖ * t ^ (-s - 1)) := ?_
+    _ ≤ (s - 1) * s * C +
+          s * ((s - 1) * ∫ (t : ℝ) in Set.Ioi T, ‖S t - l * t‖ * t ^ (-s - 1)) := ?_
+    _ ≤ (s - 1) * s * C + s * ε := ?_
```

Comment:
Same comment here. I suggest **not** indenting the `calc` steps, so they stand out visually from the proofs indented in between.